docs/content/preview/yugabyte-voyager/reference/data-migration/export-data.md

```diff
@@ -184,23 +464,122 @@ Usage: yb-voyager export data from target [ <arguments> ... ]
+```
+
+| Comma-separated list of the tables to export data. Table names can also be glob patterns containing wildcard characters, such as an asterisk (*) (matches zero or more characters) or question mark (?) (matches one character). To use a glob pattern for table names, enclose the list in single quotes ('').<br> For example, `--table-list '"Products", order*'`. |
+| --exclude-table-list |
+
+```yaml{.nocopy}
+export-data-from-target:
+  exclude-table-list:
+```
+
+| Comma-separated list of the tables to exclude during export. Table names follow the same convention as `--table-list`. |
+| --table-list-file-path |
+
+```yaml{.nocopy}
+export-data-from-target:
+  table-list-file-path:
+```
+
+| Path of the file containing the list of table names (comma-separated or line-separated) to export. Table names use the same convention as `--table-list`. |
+| --exclude-table-list-file-path |
+
+```yaml{.nocopy}
+export-data-from-target:
+  exclude-table-list-file-path:
+```
+
+| Path of the file containing the list of table names (comma-separated or line-separated) to exclude while exporting data. Table names follow the same convention as `--table-list`. |
+| -e, --export-dir |
+
+```yaml{.nocopy}
+export-dir:
+```
+
+| Path to the export directory. This directory is a workspace used to store exported schema DDL files, export data files, migration state, and a log file.|
+| --send-diagnostics |
+
+```yaml{.nocopy}
+send-diagnostics:
+```
+
+| Enable or disable sending [diagnostics](../../../reference/diagnostics-report/) information to Yugabyte. <br>Default: true<br> Accepted parameters: true, false, yes, no, 0, 1 |
+| --target-db-password |
+
+```yaml{.nocopy}
+target:
+  db-password:
+```
+
+| Password to connect to the target YugabyteDB database. Alternatively, you can also specify the password by setting the environment variable `TARGET_DB_PASSWORD`. If you don't provide a password via the CLI, yb-voyager will prompt you at runtime for a password. If the password contains special characters that are interpreted by the shell (for example, # and $), enclose the password in single quotes. |
+| [--target-ssl-mode](../../yb-voyager-cli/#ssl-connectivity) |
+
+```yaml{.nocopy}
+target:
+  ssl-mode:
+```
+
+| Determines whether an encrypted connection is established between yb-voyager and the database server; and whether the certificate of the database server is verified from a CA.
+
+For a list of valid options, refer to [SSL mode options](#ssl-mode-options). |
+
+| [--target-ssl-root](../../yb-voyager-cli/#ssl-connectivity) |
+
+```yaml{.nocopy}
+target:
+  ssl-root:
+```
+
+| Path to a file containing the target YugabyteDB SSL Root Certificate. If the target cluster has SSL enabled, this flag is required. |
+| -h, --help | — | Command line help for synchronize. |
+| -y, --yes | — | Answer yes to all prompts during the export schema operation. <br>Default: false |
+
+{{</table>}}
+
+#### SSL mode options
+
+The available SSL modes are as follows:
```

Comment:
Why isn't this all on the CLI page SSL Connectivity section? https://docs.yugabyte.com/preview/yugabyte-voyager/reference/yb-voyager-cli/#ssl-connectivity
Why repeat it here? Also, the linked section doesn't include this connector distinction, is it out of date?